CONTRIBUTING.md

```diff
@@ -690,33 +690,45 @@ query localConnections {
 curl -s -H "Content-Type:application/json" -H "Authorization: Bearer ${DTX_ACCESS_TOKEN}" http://localhost:26636/gateway/v1/graphql/schema.graphql
 ```
 
-### Invoking the Control Plane Proxy API
+### The Control and Data Plane Proxy APIs
 
-The Control Plane Proxy API allows you to make calls to the Confluent Cloud control plane, specifically for artifact and compute pool management endpoints defined by the configuration:
+The Control Plane Proxy API allows you to make calls to the Confluent Cloud control plane, specifically for artifact and compute pool management endpoints.
+The Data Plane Proxy API allows you to make calls to the Confluent Cloud control plane for the statement and catalog endpoints.
```

Comment:
I think, in this document, we should highlight that the control plane proxy is very generic, while the _data plane proxy_ is actually a proxy for Flink data plane routes.